longest_palindromic_substring/longest_palindromic_substring.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+*  Implementation of dp problem: Longest Palindromic substring
+*  Time complexity: O(n^2)
+*  Space complexity: O(n^2)
+*  n is size of input string
+*/
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+using namespace std;
+
+string longest_palindromic_substring(string test) {
+	int n = test.length();
+	vector< vector<bool> > substrings(n, vector<bool>(n, false));
+	int len_pallindrome = 1;
+	int pallindrome_begin = 0;
+
+	for (int i = 0; i < n; ++i) {  // Substrings of length 1
+		substrings[i][i] = true;
+	}
+
+	for (int i = 0; i < n - 1; ++i) {  // Substrings of length 2
+		if (test[i] == test[i + 1]) {
+			substrings[i][i + 1] = true;
+			pallindrome_begin = i;
+			len_pallindrome = 2;
+		}
+	}
+
+	for (int i = 3; i <= n; ++i) {  // Substrings of length 3
+		for (int j = 0; j < n - i + 1; ++j) {
+			int tmp = i + j - 1;
+			if (substrings[j + 1][tmp - 1] && test[j] == test[tmp]) {
+				substrings[j][tmp] = true;
+				if (i > len_pallindrome) {
+					len_pallindrome = i;
+					pallindrome_begin = j;
+				}
+			}
+		}
+	}
+	return test.substr(pallindrome_begin, pallindrome_begin + len_pallindrome);
+}
+
+
+int main() {
+	string test = "IIITV";
+	cout<<"Longest pallindromic substring is: "<<longest_palindromic_substring(test);
+	return 0;
+}
```

Comment:
Sir there is a blank line at end, if it wasn't there travis would've failed due to it. Somehow it has happened before that the blank line is not shown in PRs